CMakeLists.txt

```diff
@@ -131,10 +133,11 @@ if(DJINNI_WITH_PYTHON)
   target_include_directories(djinni_support_lib PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/djinni/cwrapper/>")
   target_sources(djinni_support_lib PRIVATE ${SRC_C_WRAPPER})
   source_group("cwrapper" FILES ${SRC_C_WRAPPER})
-
+  # these files might be delivered in future per default
```

Comment:
not sure what will change, this is why I added a "_might_" into the sentence ;-) 

But generally speaking, I mean the C wrapper, maybe it can be useful, or made to be useful, not just for python.